polling_stations/apps/data_finder/fixtures/integration_tests_addressbase.json

```diff
@@ -1,4 +1,57 @@
-[
+[    {
+        "fields": {
+            "name": "Foo Council",
+            "electoral_services_email": "",
+            "electoral_services_phone_numbers": [""],
+            "electoral_services_website": "",
+            "electoral_services_postcode": "",
+            "electoral_services_address": "",
+            "area": null,
+            "identifiers": ["X01"]
+        },
+        "model": "councils.council",
+        "pk": "FOO"
+    },
+    {
+        "fields": {
+            "name": "Example Council",
+            "electoral_services_email": "",
+            "electoral_services_phone_numbers": [""],
+            "electoral_services_website": "",
+            "electoral_services_postcode": "",
+            "electoral_services_address": "",
+            "area": null,
+            "identifiers": ["E06000023"]
+        },
+        "model": "councils.council",
+        "pk": "EXM"
+    },
+    {
+        "fields": {
+            "name": "",
+            "electoral_services_email": "info@eoni.org.uk",
+            "electoral_services_phone_numbers": [""],
+            "electoral_services_website": "http://www.eoni.org.uk/",
+            "electoral_services_postcode": "",
+            "electoral_services_address": "",
+            "area": null,
+            "identifiers": ["N09000003"]
+        },
+        "model": "councils.council",
+        "pk": "N09000003"
+    },
+    {
```

Comment:
I don't understand why this wasn't needed before, same with the example council above and the onspd entry below. This is largely why this commit is called 'begin fixing integration tests' at the moment they feel pretty brittle, and I get weird failures locally that don't show up on CI. But I don't think this PR is where to address that.